dandischema/tests/test_models.py

```diff
@@ -689,3 +692,53 @@ def test_embargoedaccess() -> None:
             )
         ]
     )
+
+
+def _non_contact_person_roles_args() -> List[List[RoleType]]:
+    """
+    Return a list of role lists that each do not contain the role of a contact person
+    """
+    return [[], [RoleType.Author, RoleType.DataCurator], [RoleType.Funder]]
+
+
+def _contact_person_roles_args() -> List[List[RoleType]]:
+    """
+    Return a list of role lists that each contains the role of a contact person
+    """
+    return [
+        role_lst + [RoleType.ContactPerson]
+        for role_lst in _non_contact_person_roles_args()
+    ]
+
+
+class TestContributor:
```

Comment:
out of curiosity, any specific reason to have a class here?

Reply:
I use that to group all the tests related to `Contributor`. Since there are other tests for testing different aspects/entities in `test_models.py`, I think it is beneficial to put all the given tests into an easily identifiable group represented by the class.